### PR TITLE
Add workload partitioning annotation tests from origin's openshift/conformance/parallel suite as informing job for SNO

### PR DESCRIPTION
**Pull request by jerpeter1**

### CODE REVIEW

File: ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh [hunk 1]
```diff
@@ -19,3 +19,33 @@ spec:
     systemReserved:
       memory: 3Gi
 EOF
+
+# Use first two cores on a single node for workload partitioning
+cat >"${SHARED_DIR}/manifest_single-node-workload-partitioning.yml" << EOF
```
Comment (imiller0) [inline ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh:24]: Does this enable workload partitioning on all e2e SNO tests or just the new ones being added?

Comment (jerpeter1) [reply ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh:24]: All SNO tests. AFAIK, the AWS SNO cluster only has two cores ( 4 virtual cores). Not sure about the bare metal SNO cluster.

Comment (imiller0) [reply ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh:24]: I think we would only want to enable workload partitioning for the tests we are adding. The number of cores needed by other tests for the platform will depend on how many/which operators are deployed. @browsell can you confirm?

Comment (ingvagabund) [reply ci-operator/step-registry/single-node/conf/e2e/single-node-conf-e2e-commands.sh:24]: From https://github.com/openshift/enhancements/blob/master/enhancements/workload-partitioning/management-workload-partitioning.md#goals:
```
Clusters built in this way should pass the same Kubernetes and OpenShift conformance and functional end-to-end tests as single-node deployments that are not isolating the management workloads.
```
So the current SNO jobs need all CPUs to comply with the enhancement goals when the conformance tests are ran.

**Comment by jerpeter1** [comment]
Split off enabling workload paritioning to another PR: https://github.com/openshift/release/pull/21548. Once that PR and the test PR is merged, this PRs tests should start passing.

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 2]
```diff
@@ -17943,6 +17943,80 @@ periodics:
...
+    org: openshift
+    repo: release
+  interval: 24h
+  labels:
+    ci-operator.openshift.io/variant: nightly-4.10
+    ci.openshift.io/generator: prowgen
+    ci.openshift.io/no-builds: "true"
+    job-release: "4.10"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-single-node-wp
```
Comment (omertuc) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:17962]: Add this between `name` and `spec`:

```yaml
  reporter_config:
    slack:
      channel: '#single-node-ci'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs>'
```

**Pull request closed without merging**